book/snippets/nostr/python/src/nip06.py

```diff
@@ -0,0 +1,65 @@
+# Leverages Python-Mnemonic package (ref implementation of BIP39) https://github.com/trezor/python-mnemonic
+from mnemonic import Mnemonic
+from nostr_protocol import Keys
+
+def nip06():
+    # ANCHOR: keys-from-seed24
+    # Generate random Seed Phrase (24 words e.g. 256 bits entropy)
+    print("\nKeys from 24 word Seed Phrase:")
```

Comment:
Just a my preference, for cleaned examples: in general, instead of use `\n`, can you use a `print()` outside the `ANCHOR` block?

Reply:
Sure, will make this consistent throughout, where possible.